### ISSUE TITLE
Figure out a way to run all kumu tests in node.js

### ISSUE THREAD

**Issue by monesga** [velentir/kumu#62]
To validate that we have achieved JS compatibility

Of course the goal for kumu to be a subset and the other way around is pretty much impractical

**Comment by joshuaevenson** [comment]
Good idea, and it shouldn't be too challenging. I'll likely have to move the text of the tests out of C directly so that they can be re-used.

**Comment by monesga** [comment]
I think we can keep the C based tests but change the helper functions and macros to emit jest tests 

That way we can keep the built in simple harness that runs anywhere but emit tests that we can use to validate everything on JS

Will mess around with this soon
